packages/admin/cms-admin/src/pages/pageTreeSelect/PageTreeSelectDialog.tsx

```diff
@@ -81,6 +81,8 @@ export default function PageTreeSelectDialog({ value, onChange, open, onClose, d
     const selectedPageId = value?.id;
     const classes = useStyles();
 
+    const pagesQuery = React.useMemo(() => createPagesQuery({}), []);
```

Comment:
Why do you create the pages query without the additional fields here? I believe this might be confusing for users to see the info tag in the pages page but not in the select dialog